public/market.json

```diff
@@ -0,0 +1,2866 @@
+[
```

Comment:
don't check this file in; this will be autogenerated by the python code in the parent dir